docs/guides/securing-apps/upgrading.adoc

```diff
@@ -0,0 +1,28 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+[[_upgrade_client_libraries]]
+<@tmpl.guide
+title="Upgrading the {project_name} Client Libraries"
+priority=540
+summary="How to upgrade the {project_name} Client Libraries">
+
+The client libraries are those artifacts:
+
+* Java admin client - Maven artifact `org.keycloak:keycloak-admin-client`
+* Java authorization client - Maven artifact `org.keycloak:keycloak-authz-client`
+* Java policy enforcer - Maven artifact `org.keycloak:keycloak-policy-enforcer`
```

Comment:
Shouldn't we mention `org.keycloak:keycloak-client-common-synced` as a library containing data representation classes and necessary common classes?

Reply:
Thanks for pointing this! Added the additional line with this library.